cfme/tests/services/test_catalog_item.py

```diff
@@ -456,3 +456,175 @@ def test_service_provisioning_email(request, appliance, catalog_item):
     provision_request.wait_for_request(method='ui')
     request.addfinalizer(provision_request.remove_request)
     assert result.validate(wait="60s")
+
+
+@pytest.mark.meta(coverage=[1746344])
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_assigned_unassigned_catalog_items():
+    """
+    Bugzilla:
+        1746344
```

Comment:
**Suggestion:**
As per uniformity in the tests, do you think blank is required just below to `Bugzilla` tag .?